### DISCUSSION TITLE
Hybrid Parallelisation in NekRS

### DISCUSSION THREAD

**Discussion by JunaidAhmed2520** [Nek5000/nekRS#570] [Q&A] [answered]
Hi, I had newly started learning nekRS and I was wondering whether it is possible to achieve hybrid parallelization in nekRS. If so, how can it be done?

**Comment by stgeke** [comment, marked as answer]
What do you mean by hybrid parallelization? Does it refer to running kernels on both the CPU and GPU? If so, the answer is no.

**Reply by JunaidAhmed2520** [reply]
Yeah, that's what I meant. Thanks for the information.
